R/slider.R

```diff
@@ -114,6 +114,25 @@ rangeInput <- function(inputId, label, min, max, value, step = 1, width = NULL,
 #' @param session The \code{session} object passed to function given to \code{shinyServer}.
 #' @param input_id The id of the input object
 #' @param value The value to be selected for the sldier (lower value if using range).
+#' @examples
+#' ## Only run this example in interactive R sessions
+#' if (interactive()) {
+#'   shinyApp(
+#'     ui = semanticPage(
+#'       p("The first slider controls the second"),
+#'       slider_input("control", "Controller:", min = 0, max = 20, value = 10,
+#'                    step = 1),
+#'       slider_input("receive", "Receiver:", min = 0, max = 20, value = 10,
+#'                    step = 1)
+#'     ),
+#'     server = function(input, output, session) {
+#'       observe({
+#'         val <- input$control
```

Comment:
is `val` needed here, eg. : `update_slider(session, "receive", value = input$control)`

Reply:
fixed